playbooks/adhoc/atomic_openshift_tutorial_reset.yml

```diff
@@ -0,0 +1,68 @@
+# This deletes *ALL* Docker images, and uninstalls OpenShift and
+# Atomic Enterprise RPMs.  It is primarily intended for use
+# with the tutorial as well as for developers to reset state.
+
+- hosts:
+    - OSEv3:children
+
+  sudo: yes
+
+  tasks:
+    - service: name={{ item }} state=stopped
+      with_items:
+        - openshift-master
+        - openshift-node
+        - openvswitch
+        - atomic-enterprise-master
+        - atomic-enterprise-node
+
+    - yum: name={{ item }} state=absent
+      with_items:
+        - openvswitch
+        - atomic-enterprise
+        - atomic-enterprise-master
+        - atomic-enterprise-node
+        - atomic-enterprise-sdn-ovs
+        - tuned-profiles-atomic-enterprise-node
+        - openshift
+        - openshift-master
+        - openshift-node
+        - openshift-sdn-ovs
+        - tuned-profiles-openshift-node
+
+    - shell: systemctl reset-failed
+      changed_when: False
+
+    - shell: systemctl daemon-reload
+      changed_when: False
+
+    - shell: find /var/lib/atomic-enterprise/openshift.local.volumes -type d -exec umount {} \; 2>/dev/null || true
+      changed_when: False
+
+    - shell: find /var/lib/openshift/openshift.local.volumes -type d -exec umount {} \; 2>/dev/null || true
+      changed_when: False
+
+    - shell: docker ps -a -q | xargs docker stop
+      changed_when: False
+
+    - shell: docker ps -a -q| xargs docker rm
+      changed_when: False
+
+    - shell:  docker images -q |xargs docker rmi
+      changed_when: False
+
+    - file: path={{ item }} state=absent
+      with_items:
+        - /var/lib/atomic-enterprise
+        - /etc/sysconfig/atomic-enterprise
+        - /etc/atomic-enterprise
+        - /etc/openshift
+        - /etc/openshift-sdn
+        - /root/.kube
+        - /etc/sysconfig/openshift
+        - /var/lib/openshift
```

Comment:
@sdodson also what about /etc/openshift-sdn, would it stay same for all products?